anvil/src/main/kotlin/trikita/anvil/Renderable.kt

```diff
@@ -0,0 +1,38 @@
+package trikita.anvil
+
+import android.app.Activity
+import android.app.Fragment
+import android.content.Context
+import android.util.AttributeSet
+
+@JvmOverloads
+fun renderable(
```

Comment:
We will drop support of Java DSL in the future, but if you started adding java interop, I recommend to mark your methods with `@JvmStatic` and add `@file:JvmName` annotation.

Reply:
It's not for interop, it's for view creation, since `RenderableView` is an android view and has 3 notorious constructors

Reply:
Can you explain this, please?
I don't see a reason why we need `@JvmOverloads` if not for Java inter-op here.

Reply:
@dector Sure.
This: https://github.com/Inkremental/anvil/blob/969f90b7313b95753cdfce152667d4ef754071c7/anvil/src/main/java/trikita/anvil/RenderableView.java#L12
This:
https://github.com/Inkremental/anvil/blob/969f90b7313b95753cdfce152667d4ef754071c7/anvil/src/main/java/trikita/anvil/RenderableView.java#L16
And this:
https://github.com/Inkremental/anvil/blob/969f90b7313b95753cdfce152667d4ef754071c7/anvil/src/main/java/trikita/anvil/RenderableView.java#L20

Those constructors will not go upon migration to Kotlin, and since `renderable` is just a factory for `RenderableView`, it needs to pipe all construction arguments through.

Reply:
Then you don't need overload annotation.
It will use default argument values when constructing RenderableView, trust me.

Reply:
Oh, you're right actually. My bad.
Will commit tomorrow

Reply:
Done.